assets/src/settings-page/analytics.js

```diff
@@ -18,59 +18,10 @@ import { Button, PanelRow, BaseControl, VisuallyHidden } from '@wordpress/compon
  */
 import { Options } from '../components/options-context-provider';
 import { AMPNotice, NOTICE_SIZE_SMALL } from '../components/amp-notice';
+import vendorConfigs from './vendor-configs';
 
 const GOOGLE_ANALYTICS_VENDOR = 'googleanalytics';
```

Comment:
do we still need this?

Reply:
Yes, This variable is used below for message.